tests/integration/internal/strings/strings_test.go

```diff
@@ -6,10 +6,6 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func Test_ValuesFileFromT(t *testing.T) {
```

Comment:
I'd leave this because this shows the reader that both this and the test below return the same values file name.

Reply:
Ok, restored it and added a comment explaining it instead.